docs/pages/guides/nextjs-starter-kit.mdx

```diff
@@ -0,0 +1,545 @@
+
+```dotenv file=".env.local"
+GITHUB_SECRET=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+GITHUB_ID=XXXXXXXXXXXXXXXXXXXX
+```
+
+GitHub authentication is now complete! Next,
+[add yourself as a user](#how-to-sign-up) to test out your authentication.
+
+### Auth0 authentication [#auth0-authentication]
+
+To use Auth0 auth, make sure you selected "Auth0 authentication" when running
+the installer (or you've set up the
+[provider manually](#add-multiple-authentication-providers)). This is how to set
+up your secret keys.
+
+1. Go to your [Auth0 Dashboard](https://manage.auth0.com/dashboard) and click
+   "Create Application".
+2. Enter an app name (e.g. `Liveblocks Starter Kit (dev)`). You'll need a new
+   app for each environment, so it's helpful to place "dev" in the name.
+3. Next, select "Single Page Web Applications", and press "Create".
+4. Copy your "Client ID" from the top of the page, and place it within
+   `.env.local` as `AUTH0_CLIENT_ID`.
+5. Click the "Settings" tab— we'll be making a number of changes here.
+6. Find the "Client Secret" input field, and copy the value into `.env.local` as
+   `AUTH0_CLIENT_SECRET`.
+7. Copy your "Domain" from the input field, add "https://" to the start, and
+   place it within `.env.local` as `AUTH0_ISSUER_BASE_URL`.
+8. Add the following to the "Allowed Callback URLs" textarea:
+   `http://localhost:3000/api/auth/callback/auth0`.
+9. Add the following to the "Allowed Logout URLs" textarea:
+   `http://localhost:3000`.
+10. Scroll to the bottom and press "Save changes".
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source
+      src="/images/docs/next-js-starter-kit/auth0-auth.mp4"
+      type="video/mp4"
+    />
+  </video>
+</Figure>
+
+`.env.local` should now contain these three lines, along with anything
+previously there:
+
+```dotenv file=".env.local"
+AUTH0_CLIENT_ID=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+AUTH0_CLIENT_SECRET=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+AUTH0_ISSUER_BASE_URL=https://XXXXXXXXXXXXXXXXXX.com
+```
+
+Auth0 authentication is now set up! Next, [add yourself a user](#how-to-sign-up)
+to test out your authentication.
+
+### How to sign up - add yourself as a user [#how-to-sign-up]
+
+We haven't set up a database, so we're temporarily using the `/data` folder
+instead. Before any user can sign in, they need to be added to `/data/users.ts`.
+Navigate there and add your details, for example, if you're signing in with
+`yourname@example.com`:
+
+```ts file="data/users.ts"
+{
+  id: "yourname@example.com",
+  name: "Your Name",
+  avatar: "https://liveblocks.io/avatars/avatar-0.png",
+  groupIds: ["product", "engineering", "design"],
+},
+```
+
+### Ready to go
+
+The Next.js starter kit is now ready to use! After setting up authentication,
+make sure to restart the dev server to see your authentication in action.
+
+```
+npm run dev
+```
+
+## Assorted info
+
+### Async Fetching and Error Handling
```

Comment:
I notice that we use a mix of sentence case and camel case in the titles. Can we harmonize it? From what I see in previous guides, it should be a sentence case. Unless there’s a reason for those ones?

```suggestion
### Async fetching and error handling
```